### PR TITLE
Remove usages of qsort

### PR DESCRIPTION
**Pull request by Learath2**
Caught my eye while doing the integrity PR.

### CODE REVIEW

File: src/engine/client/client.cpp [hunk 1]
```diff
@@ -1239,27 +1238,25 @@ const char *CClient::LoadMapSearch(const char *pMapName, SHA256_DIGEST *pWantedS
 	return pError;
 }
 
-int CClient::PlayerScoreNameComp(const void *a, const void *b)
+bool CClient::PlayerScoreNameComp(const CServerInfo::CClient &p0, const CServerInfo::CClient &p1)
```
Comment (def-) [inline src/engine/client/client.cpp:1241]: Oh, references. Fancy :D

Comment (def-) [reply src/engine/client/client.cpp:1241]: Is the name Comp still correct? Seems like it's more of a Less now.

Comment (Learath2) [reply src/engine/client/client.cpp:1241]: Well `std::sort` wants its compare function taking references and it wants it to return true if `a < b`. The ordering looked the same to me with or without the patch.

Comment (def-) [reply src/engine/client/client.cpp:1241]: Just saying, I assume a "Compare" function to be like str_comp and return -1, 0, 1. While a "Less" function should return a bool. So I suggest to rename all the Comp/Compare functions to Less

**Comment by Learath2** [comment]
I swear git is trying to make me look bad today :D

**Pull request merged**